sdk/src/test/java/com/hedera/hashgraph/sdk/TokenInfoTest.java

```diff
@@ -19,69 +19,89 @@
+        .getPublicKey();
+    private static final PublicKey testFreezeKey = PrivateKey.fromString(
+            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e13")
+        .getPublicKey();
+    private static final PublicKey testWipeKey = PrivateKey.fromString(
+            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e14")
+        .getPublicKey();
+    private static final PublicKey testSupplyKey = PrivateKey.fromString(
+            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e15")
+        .getPublicKey();
+    private static final PublicKey testFeeScheduleKey = PrivateKey.fromString(
+            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e16")
+        .getPublicKey();
+    private static final PublicKey testPauseKey = PrivateKey.fromString(
+            "302e020100300506032b657004220420db484b828e64b2d8f12ce3c0a0e93a0b8cce7af1bb8f39c97732394482538e17")
+        .getPublicKey();
+    private static final TokenId testTokenId = TokenId.fromString("0.6.9");
+    private static final AccountId testTreasuryAccountId = AccountId.fromString("7.7.7");
+    private static final AccountId testAutoRenewAccountId = AccountId.fromString("8.9.0");
+    private static final String testTokenName = "test token name";
+    private static final String testTokenSymbol = "TTN";
+    private static final String testTokenMemo = "memo";
+    private static final int testTokenDecimals = 3;
+    private static final long testTokenTotalSupply = 1000L;
+    private static final Boolean testTokenFreezeStatus = true;
+    private static final Boolean testTokenKycStatus = true;
+    private static final boolean testTokenIsDeleted = false;
+    private static final List<CustomFee> testTokenCustomFees = Arrays.asList(
+        new CustomFixedFee().setFeeCollectorAccountId(new AccountId(4322)).setDenominatingTokenId(new TokenId(483902))
+            .setAmount(10),
+        new CustomFractionalFee().setFeeCollectorAccountId(new AccountId(389042)).setNumerator(3).setDenominator(7)
+            .setMin(3).setMax(100));
+    private static final TokenType testTokenType = TokenType.FUNGIBLE_COMMON;
+    private static final TokenSupplyType testTokenSupplyType = TokenSupplyType.FINITE;
+    private static final long testTokenMaxSupply = 1000000L;
+    private static final boolean testTokenPauseStatus = true;
+    private static final LedgerId testTokenLedgerId = LedgerId.MAINNET;
+    private static final Duration testAutoRenewPeriod = Duration.ofHours(10);
+    private static final Instant testExpirationTime = Instant.ofEpochSecond(1554158542);
+
     @BeforeAll
     public static void beforeAll() {
         SnapshotMatcher.start();
     }
 
-    @AfterClass
+    @AfterAll
```

Comment:
Why do we need this change?

Reply:
@rokn newer annotation comes from newer junit (jupiter) version, but it is pretty same thing and that's it.